lib/cc/services/github_pull_requests.rb

```diff
@@ -87,8 +86,7 @@ def presenter
   def update_status_error
     update_status(
       "error",
-      "Code Climate encountered an error while attempting to analyze this " +
-        "pull request."
+      message
```

Comment:
Agreed. cc-services should not be in the business of crafting messages based on data, etc. But having a sane, generic default seems totally reasonable. And is important for something like this which is a shared, open source gem that has to think about backwards compatibility.